pandas/core/panel.py

```diff
@@ -870,7 +871,11 @@ def to_frame(self, filter_observations=True):
 
         Parameters
         ----------
-        filter_observations : boolean, default True
+        dropna : boolean, default False
+            Drop (major, minor) pairs without a complete set of observations
+            across all the items
+
+        filter_observations : boolean, default False, [deprecated]
```

Comment:
How is that possible? If you don't serve the "filter_observations" argument then you don't use the deprecated argument and use dropna instead. With that filter_observations is False implicitly.